config.js

```diff
@@ -50,6 +50,15 @@ const ConfigOptions = () => {
             stripDebug
         },
 
+        /**
+         * f-logger-related variables
+         */
+        logger: {
+            dir: 'js',
```

Comment:
Why not just have `dir`? I don't think `subDir` is needed anymore?

Reply:
PathBuilder.js requires it.

```
jsErrorLoggerSubDir: `${config.webRootDir}/${config.logger.dir}/${config.logger.subDir}`
```

Reply:
Just change `config.logger.dir` to `"js/shared"`?

Reply:
Cool updated now.